hal/src/b5som/network/quectel_ncp_client.cpp

```diff
@@ -685,21 +696,30 @@ int QuectelNcpClient::initReady() {
     }
 
     // Select (U)SIM card in slot 1, EG91 has two SIM card slots
-    if (ncpId() == PLATFORM_NCP_QUECTEL_EG91_E || ncpId() == PLATFORM_NCP_QUECTEL_EG91_NA) {
+    if (ncpId() == PLATFORM_NCP_QUECTEL_EG91_E || \
+        ncpId() == PLATFORM_NCP_QUECTEL_EG91_NA || \
+        ncpId() == PLATFORM_NCP_QUECTEL_EG91_EX) {
         CHECK_PARSER(parser_.execCommand("AT+QDSIM=0"));
     }
 
     // Enable flow control and change to runtime baudrate
-    // TODO: Uncomment when hardware is rev'd to fix CTS/RTS swap.
-    // CHECK_PARSER(parser_.execCommand("AT+IFC=2,2"));
-    CHECK(changeBaudRate(QUECTEL_NCP_RUNTIME_SERIAL_BAUDRATE));
```

Comment:
Do we need to remove this?

Reply:
It should be 
```
    // Enable flow control and change to runtime baudrate
    auto runtimeBaudrate = QUECTEL_NCP_DEFAULT_SERIAL_BAUDRATE;
    uint32_t hwVersion = 0xFF;
    hal_get_device_hw_version(&hwVersion, nullptr);
    if (hwVersion == 0x00) {
        CHECK_PARSER(parser_.execCommand("AT+IFC=0,0"));
    } else {
        runtimeBaudrate = QUECTEL_NCP_RUNTIME_SERIAL_BAUDRATE;
        CHECK_PARSER(parser_.execCommand("AT+IFC=2,2"));
    }
    CHECK(changeBaudRate(runtimeBaudrate));
```
If we want to change baudrate to 460800, we can simply change `QUECTEL_NCP_RUNTIME_SERIAL_BAUDRATE`

Reply:
Ah, sorry, I've missed it. :+1:

Reply:
Fixed